packages/spotlight/src/spotlight.js

```diff
@@ -648,8 +676,7 @@ const Spotlight = (function () {
 		// focus(<containerId>)
 		// focus(<extSelector>)
 		/**
-		 * Focuses the specified component ID, container ID, element selector, or the default
-		 * container.
+		 * Gives focus to an element.
```

Comment:
no, the param part already has the information. I just removed duplications.

Reply:
Maybe it would be better, then, to say 'Sets focus.'?  Since 'to an element' is slightly misleading given the way focus can be set to containers as well.